modules/vpc/main.tf

```diff
@@ -1049,86 +1047,21 @@ module "user_management" {
 
 #tunnel2_preshared_key
 
-resource "aws_vpn_gateway" "vpn_gateway" {
-  count = "${var.enabled * var.enable_vpn * length(var.arenas)}"
-
-  lifecycle {
-    create_before_destroy = true
-  }
-
-  vpc_id = "${element(aws_vpc.nubis.*.id, count.index)}"
-
-  tags {
-    Name             = "${var.aws_region}-${element(var.arenas, count.index)}-vpn-gateway"
-    ServiceName      = "${var.account_name}"
-    TechnicalContact = "${var.technical_contact}"
-    Arena            = "${element(var.arenas, count.index)}"
-  }
-}
-
-resource "aws_customer_gateway" "customer_gateway" {
-  count = "${var.enabled * var.enable_vpn}"
-
-  lifecycle {
-    create_before_destroy = true
-  }
-
-  bgp_asn = "${var.vpn_bgp_asn}"
-
-  ip_address = "${var.ipsec_target}"
-  type       = "ipsec.1"
-
-  tags {
-    Name             = "${var.aws_region}-customer-gateway"
-    ServiceName      = "${var.account_name}"
-    TechnicalContact = "${var.technical_contact}"
-  }
-}
-
-resource "aws_vpn_connection" "main" {
-  count = "${var.enabled * var.enable_vpn * length(var.arenas)}"
+module "vpn" {
+  source = "github.com/nubisproject/nubis-terraform-vpn?ref=develop"
 
-  lifecycle {
-    create_before_destroy = true
-  }
-
-  vpn_gateway_id      = "${element(aws_vpn_gateway.vpn_gateway.*.id, count.index)}"
-  customer_gateway_id = "${aws_customer_gateway.customer_gateway.id}"
-  type                = "${aws_customer_gateway.customer_gateway.type}"
-  static_routes_only  = false
-
-  tags {
-    Name             = "${var.aws_region}-${element(var.arenas, count.index)}-vpn"
-    ServiceName      = "${var.account_name}"
-    TechnicalContact = "${var.technical_contact}"
-    Arena            = "${element(var.arenas, count.index)}"
-  }
-}
-
-resource "aws_route" "vpn-public" {
-  count = "${var.enabled * var.enable_vpn * length(var.arenas)}"
-
-  lifecycle {
-    create_before_destroy = true
-  }
-
-  route_table_id = "${element(aws_route_table.public.*.id, count.index)}"
-
-  destination_cidr_block = "10.0.0.0/8"
-  gateway_id             = "${element(aws_vpn_gateway.vpn_gateway.*.id, count.index)}"
-}
-
-resource "aws_route" "vpn-private" {
-  count = "${3 * var.enabled * var.enable_vpn * length(var.arenas)}"
-
-  lifecycle {
-    create_before_destroy = true
-  }
-
-  route_table_id = "${element(aws_route_table.private.*.id, count.index)}"
+  enabled           = "${var.enabled * var.enable_vpn}"
+  region            = "${var.aws_region}"
+  arenas            = "${var.arenas}"
+  account_name      = "${var.account_name}"
+  technical_contact = "${var.technical_contact}"
 
-  destination_cidr_block = "10.0.0.0/8"
```

Comment:
what hapenned to destination_cidr_block ?

Reply:
Good catch, I missed this and will add it back again